salt/platform-testing/general.sls

```diff
@@ -43,6 +44,16 @@
   {%- endfor -%}
 {%- endif -%}
 
+{%- set fh_hosts = [] -%}
+{%- set fh_nodes = salt['pnda.ip_addresses']('flink') -%}
+{%- if fh_nodes is not none and fh_nodes|length > 0 -%}
```

Comment:
there would be only one history server instance. Not sure why we are having a for loop here.

Reply:
This is to filter specific roles node i.e. to filter out the nodes with role *flink* from set of nodes.